tests/unit/aroi/test_leaders_scoring.py

```
@@ -4,7 +4,49 @@
+    just confirm the leaderboard side imports the same function and
+    its output flows correctly across boundary thresholds.
+    """
+
+    def test_imports_shared_classifier(self):
+        """The leaderboard module imports the canonical classifier from
+        aroi_validation, NOT a local re-implementation."""
+        from allium.lib.aroi_validation import classify_v3_tier
+        # _classify_v3_tier_local is the imported alias used in aroileaders.
+        assert _classify_v3_tier_local is classify_v3_tier
+
+    @pytest.mark.parametrize("v3,total,expected_tier", [
```

Ruff (0.15.12)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)